package.json

```diff
@@ -1,13 +1,75 @@
 {
   "name": "@salesforce/core",
-  "version": "6.7.6",
+  "version": "6.7.7-qa.0",
   "description": "Core libraries to interact with SFDX projects, orgs, and APIs.",
   "main": "lib/index",
   "types": "lib/index.d.ts",
   "license": "BSD-3-Clause",
   "engines": {
     "node": ">=18.0.0"
   },
+  "exports": {
+    ".": {
+      "types": "./lib/index.d.ts",
+      "require": "./lib/index.js",
+      "import": "./lib/index.js",
```

Comment:
node docs says these 2 should be mutually exclusive:

> "import" - matches when the package is loaded via import or import(), or via any top-level import or resolve operation by the ECMAScript module loader. Applies regardless of the module format of the target file. Always mutually exclusive with "require".

https://nodejs.org/api/packages.html#conditional-exports

but here it's fine because sfdx-core isn't ESM-yet, right?

Reply:
it's a good question.  They're all cjs for now--if we did decide to dual-publish these, we'd have `require` and `import` pointing at different files.

It worked with both an ESM (PDR) and non-ESM (SDR) like this.  Maybe it's overkill unless we dual-publish?

Reply:
It feels a bit like overkill since there's nothing conditional about it currently and it's 60 extra lines in package.json.